spec.html

```diff
@@ -5893,15 +5903,16 @@
 
     <!-- es6num="8.2.3" -->
     <emu-clause id="sec-setrealmglobalobject" aoid="SetRealmGlobalObject">
-      <h1>SetRealmGlobalObject ( _realmRec_, _globalObj_ )</h1>
-      <p>The abstract operation SetRealmGlobalObject with arguments _realmRec_ and _globalObj_ performs the following steps:</p>
+      <h1>SetRealmGlobalObject ( _realmRec_, _globalObj_, _thisValue_ )</h1>
+      <p>The abstract operation SetRealmGlobalObject with arguments _realmRec_, _globalObj_, and _thisValue_ performs the following steps:</p>
```

Comment:
If thisValue is optional, follow the associated notational conventions (brackets around the signature). Also I think the signature is enough, the following paragraph adds nothing (it's useful for some of the old-style abstract ops that don't use the signature notation).